core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
I was under the impression it would be `ManagedServer`. You were not. In either case, we shouldn't arbitrarily choose a name without involving others. We should do a quick poll to see how people feel about the two naming options.